### DISCUSSION TITLE
Testing Community [General Information]

### DISCUSSION THREAD

**Discussion by antonkri** [eclipse-score/score#109] [Testing Community]
- Community Lead: tbd

- Participants
  - Philipp Ahmann @pahmann
  - Lukasz Tekieli, @ltekieli 
  - Daniel Klotz @dneail1601
  - Laszlo Bukai @LBukai
  - Alberto Tejada @atm-acn
  - Dragan Bjedov, @draganbjedov
  - Julian Schneider, @julian711 

- Slack channel: t.b.d.

- GitHub project: t.b.d.

- Following links can be also helpful to you:
  - [SCORE main page](https://eclipse-score.github.io/)
  - [Get Involved](https://eclipse-score.github.io/get_involved.html#get-involved) 
  - [Technical Documentation](https://eclipse-score.github.io/score/)
  - [Project Handbook](https://eclipse-score.github.io/score/platform_management_plan/project_management.html)
  - [SCORE releases & roadmap](https://eclipse-score.github.io/score/score_releases/index.html)

**Comment by dneail1601** [comment]
Rebecca Markert and Frank Scholter have to be added.
@markert-r 
@FScholPer
